src/WebJobs.Script/Extensions/TraceWriterExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Azure.WebJobs.Host;
+
+namespace Microsoft.Azure.WebJobs.Script
+{
+    public static class TraceWriterExtensions
```

Comment:
Merged them into the one under Extensions (where we keep all the other extension classes)